build.gradle.kts

```diff
@@ -11,8 +11,19 @@ allprojects {
     }
 }
 
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath ("com.google.firebase:firebase-appdistribution-gradle:2.2.0")
```

Comment:
It should be added at https://github.com/code4romania/de-urgenta-android/blob/develop/plugins/configuration/build.gradle.kts#L19-L28 :)

Reply:
@DeKaN thanks! I'll fix this :D